### DISCUSSION TITLE
Submit with Content-Type=application/json

### DISCUSSION THREAD

**Discussion by gevezex** [vueform/vueform#140] [Questions] [answered]
Is it possible to submit the form with Content-Type=application/json ? 
Because I a m getting the form data in python flask backend what can't handle GroupElements well. The keys are coming in string format like "CheckboxgroupElement[0]", "CheckboxgroupElement[1]" etc where I need to parse these keys with Regular Expressions, what is ugly.
So is it possible to post the form dat in json format?

**Comment by adamberecz** [comment, marked as answer]
You can configure axios globally in `vueform.config.js` to use the right headers:
https://vueform.com/reference/configuration#axios

I believe this should do the trick:
```js
// vueform.config.js

import axios from 'axios'

axios.defaults.headers.post = {
  'Content-Type': 'application/json'
}
// ...

export default {
  axios,
  // ...
}
```

**Reply by gevezex** [reply]
This works like a charm. Thnx!
